eng/scripts/build.ps1

```diff
@@ -1,12 +1,20 @@
 #Requires -Version 7.0
-param($filter, [switch]$vet, [switch]$generate, [switch]$skipBuild, $parallel = 5)
+param($filter, [switch]$clean, [switch]$vet, [switch]$generate, [switch]$skipBuild)
 
+$startingDirectory = Get-Location
+$root = Resolve-Path ($PSScriptRoot + "/../..")
+Set-Location $root
 $sdks = @{};
 
 foreach ($sdk in (./eng/scripts/get_module_dirs.ps1 -serviceDir 'sdk/...')) {
     $name = $sdk | split-path -leaf
     $sdks[$name] = @{
-        'path' = $sdk;
+        'path'      = $sdk;
```

Comment:
Is there an intent to eventually have sdk/local config specific values for these instead of using the ones in the global scope?

Reply:
I originally needed to do this when it was parallel because the global scope variables were empty FWICT.

Now that it is not parallel, I guess it is not needed.